package.json

```diff
@@ -37,9 +37,10 @@
     "babel-core": "^6.26.0",
     "babel-eslint": "~7.1.1",
     "babel-loader": "^6.4.1",
+    "babel-plugin-transform-async-to-generator": "~6.24.1",
     "babel-plugin-transform-regenerator": "6.26.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
+    "babel-preset-es2015": "~6.24.1",
+    "babel-preset-react": "~6.24.1",
```

Comment:
What was wrong with allowing minor releases as well as long as they stay compatible?

Reply:
Probably nothing if tested and reviewed in changelog. 
But this would be an exception when looking at the rest of the listed dependencies - they are either exact or patch-flexible (with `~`).

Reply:
Indeed we want `"dependencies"` to be stable (and we have a bot to try and update them). `"devDependencies"` can be a bit more liberal, I think. I won't dwell on it, I just wondered if there was an explicit reason to change it.